packages/assistant/src/RPA/Assistant/date_picker.py

```diff
@@ -0,0 +1,87 @@
+from flet import Dropdown, UserControl, Container, Row, Icon, PopupMenuButton, PopupMenuItem, Text
+from flet.dropdown import Option
+from flet.icons import CALENDAR_MONTH, ARROW_BACK_ROUNDED, ARROW_FORWARD_ROUNDED
+from flet import MainAxisAlignment
+from datetime import datetime, date
+from calendar import monthrange
+
+
+class DatePicker(UserControl):
+
+    def _year_increment(self, e):
+        self.selected_year += 1
+        self.header_element[1].value = self.selected_year
+        self.update()
+
+    def _year_decrement(self, e):
+        self.selected_year -= 1
+        self.header_element[1].value = self.selected_year
+        self.update()
+    
+    def _select_month(self, e):
+        months = (("Jan", "Feb", "Mar", "Apr"), ("May", "Jun", "Jul", "Aug"), ("Sep", "Oct", "Nov", "Dec"))
```

Comment:
Q: Why months are grouped like in 3 kind of quarters?

Reply:
this is also for displaying purposes, they will be displayed 4 on a row in 3 rows